bin/ch/jstoc.py

```diff
@@ -4,15 +4,15 @@
 # Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 #-------------------------------------------------------------------------------------------------------
 
-from __future__ import with_statement # py 2.5
```

Comment:
https://docs.python.org/2/library/__future__.html

`from __future__ import ...` should be a no-op in Python 3 and in order to use the print function you'll need to have at least the following:

```
from __future__ import print_function
```
I'm assuming the existing import is necessary to work in python 2.

So probably you'd want to do:

```py
from __future__ import print_function, with_statement
```

I think the operative change here which makes things compatible in both python 2 and python 3 is changing the print keyword to the print function.